src/NexusMods.DataModel/Synchronizer/Synchronizer.sql

```diff
@@ -131,8 +141,9 @@ SELECT
 FROM
   synchronizer.LeafLoadoutItems (db) loadout_item
   LEFT JOIN MDB_LOADOUTITEMGROUPPRIORITY (Db => db) priority ON priority.Target = loadout_item.Parent
+  LEFT JOIN synchronizer.LoadoutGroups (db) loadout_item_group ON loadout_item_group.Id = loadout_item.Parent
 WHERE
-  priority.Id IS NOT NULL
+  priority.Id IS NOT NULL AND loadout_item.IsEnabled AND loadout_item_group.IsEnabled
```

Comment:
`synchronizer.LeafLoadoutItems` already has the enabled state, we should use that one instead